specification/paths/ConsolidateShipments.json

```
@@ -0,0 +1,56 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "experimental"
+        ]
+      }
+    ],
+    "summary": "Update given shipments with the provided consolidation shipment id.",
+    "description": "This endpoint updated provided shipment ids with the consolidation shipment id.",
+    "requestBody": {
+      "description": "The consolidation shipment id and the shipment ids.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "properties": {
+              "data": {
+                "properties": {
+                  "consolidation_shipment_id": {
+                    "$ref": "#/components/schemas/Uuid"
+                  },
+                  "shipments": {
```

I know this is the proposed structure in the ticket, but all RPC endpoints have attributes ending with `_id` so let's do this here as well.
```suggestion
                  "shipment_ids": {
```